packages/babel-types/src/index.js

```diff
@@ -398,6 +404,7 @@ export function inherits(child, parent) {
   }
 
   t.inheritsComments(child, parent);
+  traverse.copyCache(parent, child);
```

Comment:
also the only use of `babel-traverse` in `babel-types` (caused an issue when updating to babel 6 since both were deps of each other)